lib/chef/application/client.rb

```diff
@@ -298,6 +298,15 @@ class Chef::Application::Client < Chef::Application
     description: "Use cached cookbooks without overwriting local differences from the server",
     boolean: false
 
+  option :target,
+    short: "-t TARGET",
+    long: "--target TARGET",
+    description: "Target the Chef Client run against a remote system or device",
```

Comment:
This should utilize the branding values we merged in

Reply:
done.